packages/node-cli/src/utils/AuthenticationUtils.js

```diff
@@ -160,5 +174,43 @@ async function authenticateCi(params, sdkPath, projectFolder, executionEnvironme
 		.withCommandParameters(authenticateCiExecutionContext.getParams())
 		.build();
 }
+/**
+ * 
+ * @param {String} authid 
+ * @param {String} sdkPath 
+ * @param {ExecutionEnvironmentContext} executionEnvironmentContext 
+ * @returns {SdkOperationResult}
+ */
+async function checkIfReauthzorizationIsNeeded(authid, sdkPath, executionEnvironmentContext) {
+	const sdkExecutor = new SdkExecutor(sdkPath, executionEnvironmentContext);
+	const inspectAuthContext = SdkExecutionContext.Builder.
+		forCommand(COMMANDS.INSPECT_AUTHORIZATION.SDK_COMMAND)
```

Comment:
Please put the dot "." from the previous line here so make it uniform with the other method calls below this.

Reply:
Sure. I didn't realize about it. I'll change it.